assets/collapse-sections.js

```diff
@@ -0,0 +1,73 @@
+// container: querySelector for div to apply sections to
+// hidden: (optional) whether the sections should be collapsed by default. Default false.
+function sections(container, hidden) {
+	if (hidden == undefined) hidden = false;
+	// Get all the <h2> and <h3> headings in the container
+	const headings = document.querySelectorAll(container + ' h2,' + container + ' h3');
+	
+	Array.prototype.forEach.call(headings, heading => {
+	  // Give each <h2> a toggle button child
```

Comment:
It looks like this file mixes tabs and 2-indent spaces... please convert all of this to tabs